### ISSUE TITLE
TypeError: Cannot read property 'then' of undefined at Scope.$scope.save

### ISSUE THREAD

**Issue by benderunit** [Urigo/angular-meteor#859]
With the latest version of angular-meteor (1.2.1) I can't use calling save().then(...).

I get an error message in the console:

```
TypeError: Cannot read property 'then' of undefined
    at Scope.$scope.save
```
I've tried to use the classic save and cancel buttons like in step 6 of the tutorial
http://angular-meteor.com/tutorials/angular1/bind-one-object

**Comment by amadeus-m** [comment]
I have same issue. I guess the keyword 'return' in the file /packages/angular-meteor-data/modules/angular-meteor-object.js line 62 is not necessary

**Comment by isibz** [comment]
I have the same issue

**Comment by Urigo** [comment]
Released in 1.2.2 , thanks @DAB0mB
